apps/dialtone-documentation/docs/utilities/backgrounds/repeat.md

```diff
@@ -7,14 +7,14 @@ description: Utilities for controlling if or how an element's background image r
 
 Use `d-bgr-{n}` to how an element's background image repeats.
 
-<code-well-header class="d-fl-col4 d-fw-wrap d-g12 d-p12 d-bgc-green-100 d-bgo50" custom>
+<code-well-header class="d-fl-col3 d-fw-wrap d-g16 d-p16 d-bgc-secondary" custom>
   <div class="d-d-flex d-fd-column d-ai-center d-stack4" v-for="i in repeat">
       <div
-        class="d-fl-center d-w128 d-h128 d-bgc-green-200 d-bar8 d-bc-purple-200 d-of-hidden d-bgp-tl d-bgs-var" style="--bgg-size: 65% 65%; background-image: url('https://cdn.jpegmini.com/user/images/slider_puffin_before_mobile.jpg');"
+        class="d-fl-center d-w128 d-h128 d-bgc-moderate d-bar8 d-of-hidden d-bgp-tl d-bgs-var" style="--bgg-size: 65% 65%; background-image: url('https://4.bp.blogspot.com/-EVbXg5iW6qY/ULcKZEC-bnI/AAAAAAAACCI/kZDtjeKwQlo/s1600/puffin1.jpg');"
```

Comment:
do we want to store this image locally instead of an external link? This link could possibly break at any time.

Reply:
updated